main.go

```diff
@@ -61,7 +61,7 @@ func lastWinHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	teamname = matchedTeamNames[0]
-	team := allTeams[teamname]
+	team := allTeams[strings.ToLower(teamname)]
 	espnLink := fmt.Sprintf(urls[team.Division]["espn"], team.ID)
 	imgLink := fmt.Sprintf(urls[team.Division]["img"], team.ID)
 	if entry, ok := cache[teamname]; ok && time.Since(entry.lastRefresh) < ttl {
```

Comment:
tbh not sure if teamname needs to be lower case for the cache or if the key in the map needs to be lower case in the first place

Reply:
fine either way? i guess more consistent if both store in the same case